runtime/criusupport/criusupport.cpp

```diff
@@ -209,6 +209,63 @@ toggleSuspendOnJavaThreads(J9VMThread *currentThread, BOOLEAN suspend)
 	}
 }
 
+static UDATA
+notCheckpointSafeFrameWalkFunction(J9VMThread *vmThread, J9StackWalkState *walkState)
+{
+	J9Method *method = walkState->method;
+	if (NULL != method) {
+		J9ClassLoader *methodLoader = J9_CLASS_FROM_METHOD(method)->classLoader;
+		/* we only enforce this in methods loaded by the bootloader */
+		if ((NULL == methodLoader) || (methodLoader == vmThread->javaVM->systemClassLoader)) {
```

Comment:
Not important, but I don't believe this field can ever be NULL. Certainly most (if not all) other users aren't checking it.